var/spack/repos/builtin/packages/py-genshi/package.py

```diff
@@ -9,10 +9,10 @@
 class PyGenshi(PythonPackage):
     """Python toolkit for generation of output for the web"""
     homepage = "https://genshi.edgewall.org/"
-    url      = "http://ftp.edgewall.com/pub/genshi/Genshi-0.7.tar.gz"
+    url = "https://github.com/edgewall/genshi/archive/refs/tags/0.7.7.tar.gz"
+    git = "https://github.com/edgewall/genshi.git"
 
-    version('0.7', sha256='1d154402e68bc444a55bcac101f96cb4e59373100cc7a2da07fbf3e5cc5d7352')
-    version('0.6.1', sha256='fed947f11dbcb6792bb7161701ec3b9804055ad68c8af0ab4f0f9b25e9a18dbd')
-    version('0.6', sha256='32aaf76a03f88efa04143bf80700399e6d84eead818fdd19d763fd76af972a4b')
+    version('master', branch='master')
+    version('0.7.7', sha256='53920c8ed890a3ce0e274845abe82d34736191ffc51300788d4cc71e4fcdd0eb')
 
-    depends_on("py-setuptools@:57", type='build')
+    depends_on("py-setuptools", type='build')
```

Comment:
Missing dep on six

Reply:
Thank you for digging that up!  I read the versionless `setuptools` requirement in Genshi's `pyproject.toml` but missed the `install_requires = six` in `setup.cfg`.  Which I guess makes sense because one is a build requirement and the other is also an install requirement.

Many thanks for your careful reading and comments!